src/globalComponents/Navbar.js

```diff
@@ -39,12 +39,47 @@ class Navbar extends React.Component<Props, State> {
 	constructor(props: Props) {
 		super(props);
 
+		// Get initial money from cookies.
+		const cookies = new Cookies();
 		this.state = {
-			userName: "FRICK", // NEED API CALL FOR CURRENT USER
-			userCurrency: 145555 // NEED API CALL FOR USER CURRENCY
+			userName: cookies.get('userName'),

```

Comment:
Did you test that this works when there are no cookies set?

Reply:
I'll try that now.

Reply:
![image](https://user-images.githubusercontent.com/42626045/76548276-df0e8780-6464-11ea-8ef6-7f3eb5b5c70d.png)

Here's the error I get when no cookie is set.